### ISSUE TITLE
Allow multiple language based searching for repositories

### ISSUE THREAD

**Issue by rajdip-b** [max-programming/hacktoberfest-projects#108]
Currently, the site only allows us to search repositories by single language. A support to search by multiple languages might be a great feature.

**Comment by caycun** [comment]
@max-programming could you assign this issue for me? I'd love to work on it. 😃

**Comment by max-programming** [comment]
Yeah this is an important enhancement! You can take this up @caycun

**Comment by HemanthTigga** [comment]
@max-programming  can i also work on the issue? I'd love to work on it.

**Comment by max-programming** [comment]
@HemanthTigga The author is already working on it. Two people working on the same thing will result in same PRs which makes things difficult to manage. Maybe after he does it, you can see what that lacks and enhance it

**Comment by HemanthTigga** [comment]
Surely.

**Comment by caycun** [comment]
@HemanthTigga the multiple searching is done, but it needs a feature where if you click the language that is already chosen it should be removed. I will make a PR later. Would you like to work on that feature?

**Comment by HemanthTigga** [comment]
Sure I will work on this feature 😃

**Comment by max-programming** [comment]
@HemanthTigga Create a new issue for it once @caycun is done ✌️

**Comment by Anurag127001** [comment]
@max-programming I would be glad to help if u would assign me

**Comment by max-programming** [comment]
I think probably @caycun is busy with something else
Let us know if you're still up for this Cayle

**Comment by max-programming** [comment]
@Anurag127001 Are you still up for this?
